alerts/openshift-virtualization-operator/WaspNamespaceDetected.md

```diff
@@ -0,0 +1,45 @@
+# WaspNamespaceDetected
+
+## Meaning
+
+This alert fires when there is a duplicate wasp-agent deployment on the cluster
```

Comment:
Please provide more details about what wasp is.

Reply:
Fixed